src/module.c

```diff
@@ -33,8 +33,8 @@
  * The comments in this file are used to generate the API documentation on the
  * Redis website.
  *
- * Each function starting with RM_ and preceded by a block comment is included
- * in the API documentation. To hide an RM_ function, put a blank line between
+ * Each function starting with VK_ and preceded by a block comment is included
+ * in the API documentation. To hide an VK_ function, put a blank line between
```

Comment:
```suggestion
 * in the API documentation. To hide a VK_ function, put a blank line between
```

Isn't english great.

Reply:
Not VM for ValkeyModule?

Reply:
You are right. VM is closer to RM. Will update